app/config/collections.php

```diff
@@ -2875,6 +2875,16 @@
                 'required' => true,
                 'array' => true,
             ],
+            [
```

Comment:
Is it a new field? What did we use before?

Reply:
it's a new field, previously we only had it on file as `algorithm` attribute.